cartographer_ros_msgs/srv/StartTrajectory.srv

```diff
@@ -12,8 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cartographer_ros_msgs/TrajectoryOptions options
-cartographer_ros_msgs/SensorTopics topics
+string configuration_directory
+string configuration_basename
+bool use_initial_pose
+geometry_msgs/Pose initial_pose
+int32 to_trajectory_id
```

Comment:
One small note to the final reviewer: we thought that `relative_to_trajectory_id` could be a better name here... `to_trajectory_id` could be confused to be a desired trajectory ID....

Reply:
I agree. `to_trajectory_id` is indeed confusing. I am all in favor for changing this to`relative_to_trajectory_id` and also for changing the relative_pose to smth. like `trajectory_T_agent` (if it holds that `p_traj = trajectory_T_agent * p_agent`. Does not have to be in this PR though.